cloud/const.go

```diff
@@ -0,0 +1,31 @@
+// cloud package is used to hold cloud provider information that is needed
```

Comment:
I think this should be moved under goaviatrix

Reply:
My only issue with that was that `goaviatrix.AWS` is longer/uglier than `cloud.AWS`. But it does save on creating a whole new package just for constants. I will make the change.

Reply:
What if you import it as goaviatrix/cloud/ ?

Reply:
Yes, I think so too. This is basically for our internal use.

Reply:
I like that compromise. 

Moved cloud/const.go to goaviatrix/cloud/const.go, this avoids a new package in the base directory and lets us use `cloud.AWS` in the code. 👍